models/articles.model.js

```diff
@@ -0,0 +1,21 @@
+const db = require('../db/connection');
+
+
+
+exports.selectArticleById = (article_id) => {
+    let sqlQuery = 'SELECT * FROM articles';
+    const queryValues = [];
+
+    if (article_id) {
+        sqlQuery += ' WHERE article_id = $1';
+        queryValues.push(article_id);
```

Comment:
There will always be an article_id for this endpoint, otherwise this model would not be triggered! The query therefore should always include the WHERE part and shouldn't be conditional 👍